projects/components/src/navigation/navigation-list.component.ts

```diff
@@ -29,7 +38,12 @@ import { FooterItemConfig, NavItemConfig, NavItemLinkConfig, NavItemType } from
             <hr *ngSwitchCase="'${NavItemType.Divider}'" class="nav-divider" />
 
             <ng-container *ngSwitchCase="'${NavItemType.Link}'">
-              <ht-nav-item [config]="item" [active]="item === activeItem" [collapsed]="this.collapsed"></ht-nav-item>
+              <ht-nav-item
+                (navItemSelected)="onNavItemSelected($event)"
+                [config]="item"
+                [active]="(this.activeItem$ | async)?.matchPaths[0] === item.matchPaths[0]"
```

Comment:
Why compare by matchpaths rather than item?

Reply:
It's still unique, and because `item` is coming from `this.navItems$` now (instead of `this.navItems`), the equality fails _even_ when `this.activeItem$` uses `this.navItems$` combined with `navigation$` as its source. I believe this is because `resolveNavItemConfigTimeRanges` uses the spread operator which returns a new object with that a different object reference.

Reply:
so first off, we definitely don't want to have both `this.navItems` and `this.navItems$` defined. We should remove the sync version and only use the async if they're different.

The failed equality when using navItems$ hints at another issue. I would guess navItems$ is not shared, so it's evaluating twice and producing different objects via spread, as you mentioned. Fix that, and we should be able to resolve active item via normal equality.

Reply:
done